File: userManager/cPlusPlus/account.cpp
#include "account.h"
#include <iostream>
using namespace std;

static int accountCounter = 0;

// construct, object, null,
account::account()
{
    setAccountCounter();
}

// construct, object, parameters, 
account::account(std::string username, std::string password)
{
    this->username = username;
    this->password = password;
    setAccountCounter();
}

// access, object, parameters,
std::string account::getUsername()
{
    return username;
}
std::string account::getPassword()
{
    return password;
}
// access, class, parameter,
int account::getAccountCounter()
{
    return accountCounter;
}

// update, class, parameters,
void account::setAccountCounter()
{
    accountCounter++;
}
// update, object, parameters,
void account::setUsername(std::string username)
{
    this->username = username;
}
void account::setPassword(std::string password)
{
    this->password = password;
}
// update, object, printer,
void account::setPrinter()
{
    std::cout << getUsername() << endl;
    std::cout << getPassword() << endl;
    std::cout << getAccountCounter() << endl;
}

File: userManager/cSharp/Account.cs
using System;
public class Account
{
    // variables, declare, initialise, 
    public string Username = "";
    public string Password = "";
    private static int AccountCounter = 0;

    // construct, object, null,
    public Account()
    {
        GetAccountCounter();
    }

    // construct, object, parameters,
    public Account(string Username, string Password)
    {
        this.Username = Username;
        this.Password = Password;
        GetAccountCounter();
    }

    // access, object, parameters,
    public string GetUsername()
    {
        return this.Username;
    }
    public string GetPassword()
    {
        return this.Password;
    }
    public int GetAccountCounter()
    {
        return AccountCounter;
    }

    // update, object, parameters,
    public void SetUsername(string Username)
    {
        this.Username = Username;
    }
    public void SetPassword(string Password)
    {
        this.Password = Password;
    }
    // update, class, parameter
    private static void SetAccountCounter()
    {
        AccountCounter++;
    }
    // update, object, printer,
    public void SetPrinter()
    {
        Console.WriteLine(GetUsername());
        Console.WriteLine(GetPassword());
        Console.WriteLine(GetAccountCounter());
    }
}